src/main/scala/org/broadinstitute/dsde/firecloud/service/NihService.scala

```diff
@@ -274,6 +274,9 @@ class NihService(val samDao: SamDAO,
     } yield ()
   }
 
+  private def allowedNihMembers(members: Set[WorkbenchEmail]): Set[WorkbenchEmail] =
+    members.filterNot(email => FireCloudConfig.Nih.denyEmailPatterns.exists(_.matches(email.value)))
+
```

Comment:
recommendation, feel free to do in a follow-on PR: log when a user is denied because of this block list so we can follow up with some record of the blocklist behaving

Reply:
good idea